File: 0070-climbing-stairs/0070-climbing-stairs.cpp
class Solution {
public:
    int climbStairs(int n) {
        int first = 0, second = 1, result = 0, steps = 0;
        while(steps < n) {
            result = first + second;
            first = second;
            second = result;
            steps++;
        }
        return result;
    }
};

File: 0070-climbing-stairs/0070-climbing-stairs.java
class Solution {
    public int climbStairs(int n) {
        int first = 0, second = 1, temp;
        for(int i = 0; i < n; ++i) {
            temp = first;
            first = second;
            second += temp;
        }
        return second;
    }
}